src/components/views/elements/Pill.js

```diff
@@ -155,7 +155,7 @@ const Pill = createReactClass({
         this.setState({resourceId, pillType, member, group, room});
     },
 
-    componentWillMount() {
+    componentDidMount() {
         this._unmounted = false;
         this._matrixClient = MatrixClientPeg.get();
         this.componentWillReceiveProps(this.props);
```

Comment:
this should be fine but does feel nasty calling a `will` method from a `did` method

Reply:
see code quality pr